src/markdown-pages/build-apps/create-an-ab-test-application/catalog-info.mdx

```diff
@@ -38,13 +38,13 @@ cd nru-programmability-course/describe-app/ab-test
 
 <Step>
 
-Create the _catalog_ directory:
+Create the _catalog_ directories:
 
 ```sh
 nr1 create --type catalog
 ```
 
-This creates a _catalog_ directory with template files for inputting custom information about your app.
+This creates a root _catalog_ directory and a _catalog_ directory for each Nerdpack item with template files for inputting custom information about your app.
```

Comment:
> ...a _catalog_ directory for each Nerdpack item with template files for inputting custom information...

Elsewhere, it sounded like these Nerdpack item-level catalog directories only hold screenshots and not template files. Is that right?

Reply:
Well spotted! This would be good to specify. 

You're right, at present, the only utility of the Nerdpack item directories is to provide a place where screenshots can be put and uploaded via `nr1 catalog:submit`. The folder structure is always `<nerdpackItem>/catalog/screenshots/`. Because it only contains a `screenshots/` folder, the `catalog/` directory in a Nerdpack item is more empty that the root level `catalog/` directory. 

That root level directory contains the template files this line is referring to, and the Nerdpack items directories do not.

We decided to mirror the root `catalog/` directory folder structure because in future we may indeed add template files to the Nerdpack item directories—room to grow.

How about something like either of the below?

[Less descriptive]
> This creates a root _catalog_ directory with template files for inputting custom information about your app, as well as a _catalog_ directory for each Nerdpack item.

[More descriptive]
> This creates a root _catalog_ directory with template files for inputting custom information about your app and a _catalog_ directory for each Nerdpack item where screenshots can be stored.

Reply:
Thanks @tyreer! I like the more descriptive suggestion.

Reply:
I like this a lot. Thanks!